content/open-source/sigstore/_index.md

```diff
@@ -13,9 +13,10 @@ Sigstore offers a new standard for signing, verifying and protecting software
 | ------------------------------- | ------------------------------------------------------------------------- |
 | [Cosign](cosign) | [How to Sign an SBOM with Cosign](cosign/how-to-sign-an-sbom-with-cosign/) |
 | [Fulcio](fulcio) | [How to Inspect and Verify Fulcio Certificates](fulcio/how-to-inspect-and-verify-fulcio-certificates/) |
+| [Policy Controller](policy-controller) | [How to Install Sigstore Policy Controller](policy-controller/how-to-install-policy-controller) |
```

Comment:
I would maybe move this under Cosign to be consistent with the way it is on [Sigstore docs](https://docs.sigstore.dev/), what do you think?

Reply:
Ah I see how that's arranged there, will move it up a spot, keen 👀 !